### PR TITLE
Feature/bbb 710 whats your national insurance number

### PR DESCRIPTION
**Pull request by aliashik-vt**

### CODE REVIEW

File: src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/NinoController.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import uk.gov.dft.bluebadge.webapp.citizen.controllers.journey.StepDefinition;
+import uk.gov.dft.bluebadge.webapp.citizen.model.Journey;
+import uk.gov.dft.bluebadge.webapp.citizen.model.form.NinoForm;
+
+@Controller
+@RequestMapping(Mappings.URL_NINO)
+public class NinoController implements StepController {
+
+  public static final String TEMPLATE = "nino";
+  public static final String FORM_REQUEST = "formRequest";
```
Comment (mgilgar) [inline src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/NinoController.java]: FORM_REQUEST is already defined in StepController, you can remove this declaration.

File: src/test/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/NinoControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import uk.gov.dft.bluebadge.webapp.citizen.model.form.NinoForm;
+
+public class NinoControllerTest {
+
+  private MockMvc mockMvc;
+  private NinoController controller;
+  @Mock private RouteMaster mockRouteMaster;
+
+  private Journey journey;
+  private String nino = "NS123456A";
```
Comment (mgilgar) [inline src/test/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/NinoControllerTest.java]: Make them:
private static final String NINO =
private static final String INVALID_NINO =

File: src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/NinoController.java (hunk 1 above)
Comment (aliashik-vt) [reply src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/NinoController.java]: I don't see FORM_REQUEST defined inside StepController

Comment (mgilgar) [reply src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/NinoController.java]: That's true, it is in Journey. Not sure if I like it but it is everywhere. It is fine then.

**Pull request merged**